sdk/tables/azure-data-tables/tests/test_encoder.py

```diff
@@ -484,13 +484,14 @@ def test_encoder_create_entity_atypical_values(self):
             "Data@odata.type": "Edm.String",
         }
         verification = json.dumps(expected_entity)
-        with pytest.raises(VerificationSuccessful):
-            client.create_entity(
-                test_entity,
-                verify_payload=verification,
-                verify_url="/foo",
-                verify_headers={"Content-Type":"application/json;odata=nometadata"}
-            )
+        # TODO: This changes between Python 3.10 and 3.11
```

Comment:
Do you mean it will get different error/ behaves differently in Python 3.10 and 3.11?